app/lib/tcomb-form/factories/LocalImageFactory.js

```diff
@@ -0,0 +1,33 @@
+import t from 'tcomb-form-native';
+import ImagePicker from 'react-native-image-picker';
+
+import localImageTemplate from '../templates/LocalImageTemplate';
```

Comment:
Please name same as default export (import in this case).
(`local...` => `Local...`)